on-chain-metrics.md

```diff
@@ -124,4 +124,114 @@ List of metrics with 5 minutes minimal resolution:
 - age_destroyed
 - exchange_inflow
 - exchange_outflow
-- exchange_balance
\ No newline at end of file
+- exchange_balance
+
+### Sentiment
+
+- sentiment_volume_consumed_telegram,
+- sentiment_volume_consumed_reddit,
+- sentiment_volume_consumed_bitcointalk,
+- sentiment_volume_consumed_twitter,
+- sentiment_volume_consumed_discord,
+- sentiment_volume_consumed_professional_traders_chat,
+- sentiment_volume_consumed_total,
+- sentiment_positive_telegram,
+- sentiment_positive_reddit,
+- sentiment_positive_bitcointalk,
+- sentiment_positive_twitter,
+- sentiment_positive_discord,
+- sentiment_positive_professional_traders_chat,
+- sentiment_positive_total,
+- sentiment_negative_telegram,
+- sentiment_negative_reddit,
+- sentiment_negative_bitcointalk,
+- sentiment_negative_twitter,
+- sentiment_negative_discord,
+- sentiment_negative_professional_traders_chat,
+- sentiment_negative_total,
+- sentiment_balance_telegram,
+- sentiment_balance_reddit,
+- sentiment_balance_bitcointalk,
+- sentiment_balance_twitter,
+- sentiment_balance_discord,
+- sentiment_balance_professional_traders_chat,
+- sentiment_balance_total
+
+### Bitmex
+
+- bitmex_perpetual_open_value,
+- bitmex_perpetual_open_interest,
+- bitmex_perpetual_basis,
+- bitmex_perpetual_funding_rate
+
+# Change Over Time
+
+- dev_activity_change_30d,
+- active_addresses_24h_change_30d,
+- active_addresses_24h_change_7d,
+- active_addresses_24h_change_1d,
+- price_usd_change_30d,
+- price_usd_change_7d,
+- price_usd_change_1d,
+- volume_usd_change_30d,
+- volume_usd_change_7d,
+- volume_usd_change_1d
+
+### Dev Activity
+
+- dev_activity
+
+### Stock to Flow
+
+- stock_to_flow
+
+### Supply Outside Exchanges
+
+- supply_outside_exchanges
+
+### Amount in Top Holders
+
+- amount_in_top_holders
```

Comment:
Can you revisit the list and groupings?
- Why `supply_on_exchanges` is removed? There is no reason to have a separate `Supply Outside Exchanges` group just to put one metric there. 
- There are more top holders metrics. Again - no point in having a group with the same name as the single metric inside it. We have amount in exchange, non_exchange holders, holder distributions, etc. etc.